seed/migrations/0198_ubid_sql_functions.py

```diff
@@ -0,0 +1,527 @@
+# Generated by Django 3.2.19 on 2023-05-15 17:38
```

Comment:
Can you explain whats going on here? 😅

Reply:
Sure, PNNL has developed some reusable functions to calculate how similar two ubids are, known as the jaccard index. With a DB migration these functions become avaliable in a db SQL command. We want to use their function `UBID_CodeArea_Jaccard`, however that function requires other functions, which require others... thus we have a rather large migration.

Ideally, we could have a text file with the sql commands and import it into the migration file. However, I ran into issues getting that to work.

References: 
https://github.com/pnnl/buildingid-plpgsql/blob/master/buildingid_functions.sql
https://github.com/pnnl/buildingid-plpgsql/blob/master/buildingid_functions.sql